File: angular/src/app/main/manufacturing/shared/dto/mfwcm.dto.ts
import { CreateOrEditMFWCRESDto } from './mfwcres.dto';
import { CreateOrEditMFWCTOLDto } from './mfwctol.dto'
export class CreateOrEditMFWCMDto {
    id:number;
    wCID: string;
    wCESC: string;
    tOTRSCCOST?: number;
    tOTTLCOST?: number;
    cOMMENTS: string;
    audtUser: string;
    audtDate?: string;
    createdBy: string;
    createDate?: string;
    resDetailDto: CreateOrEditMFWCRESDto[] = new Array<CreateOrEditMFWCRESDto>();
    toolDetailDto: CreateOrEditMFWCTOLDto[] = new Array<CreateOrEditMFWCTOLDto>();
}

File: aspnet-core/src/ERP.Application.Shared/Manufacturing/SetupForms/Dtos/MFWCMDto.cs
using System;
using Abp.Application.Services.Dto;

namespace ERP.Manufacturing.SetupForms.Dtos
{
    public class MFWCMDto : EntityDto
    {
        public string WCID { get; set; }

        public string WCESC { get; set; }

        public double? TOTRSCCOST { get; set; }

        public double? TOTTLCOST { get; set; }

        public string COMMENTS { get; set; }

        public string AudtUser { get; set; }

        public DateTime? AudtDate { get; set; }

        public string CreatedBy { get; set; }

        public DateTime? CreateDate { get; set; }

    }
}